packages/driver/src/cross-origin/communicator.ts

```diff
@@ -196,6 +201,11 @@ export class SpecBridgeCommunicator extends EventEmitter {
       data = preprocessSnapshotForSerialization(data as any)
     }
 
+    // SNAPSHOT_EVENT_PREFIX events are namespaced per primary log
+    if (eventName.includes(SNAPSHOT_EVENT_PREFIX) && !Cypress._.isEmpty(data)) {
```

Comment:
Same question about combining the if's.

Reply:
I think we can. I am also wondering if we can just use the same prefix in both areas as well